### ISSUE TITLE
YO LUCKYDOG SOMEONE FINDED THE SOURCE CODE!

### ISSUE THREAD

**Issue by JerielGT** [luckydog7/Funkin-android#943]
https://drive.google.com/file/d/1Jf2RfBzL-pT0E_NuCiLm5abd1rc57tsQ/view

**Comment by DabKnight** [comment]
> https://drive.google.com/file/d/1Jf2RfBzL-pT0E_NuCiLm5abd1rc57tsQ/view

Wait the legit one?!

**Comment by JerielGT** [comment]
Ye

**Comment by JerielGT** [comment]
Not lying and theres a port in game banana with the cutscenes and everything ported from the newgrounds too

**Comment by JoseGithubFNF** [comment]
YEEEEEEEEEEEEEEEEEEESSSSSSSSSSSSSSS!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by Rs232k** [comment]
No, itsy compiled game, not source code. Check what you posting.

**Comment by JerielGT** [comment]
Well is the game banana i think cuz that versión have all of the newgrounds versión week 7 and cutscenes..

**Comment by JerielGT** [comment]
So check what are you saying

**Comment by Tysonhutchinson** [comment]
The code has already been released but week 7 isn't on itch.io me and a few others talked to each other it wasn't any the developers so we don't know yet also I'm not one either, but we think when week 7 is out on itch.io it will be on GitHub.

**Comment by PauloBloxian** [comment]
I think you need to spam in issues lol

**Comment by PauloBloxian** [comment]
That's  is good
